source/_integrations/seventeentrack.markdown

```diff
@@ -57,6 +57,13 @@ show_delivered:
 
 Use the following templated Markdown card to list all packages in transit along their status:
 
+Properties that can be using in Lovelace markdown:
```

Comment:
This line + list does now interrupt the above sentence that ends with `:`. Additionally, code or keywords should be code fenced using a backtick, not using quotes. Lastly, I'm unsure what you mean by properties?